src/app/modules/shared/feature-toggles/filters/targeting/targeting-feature-filter.model.ts

```diff
@@ -0,0 +1,18 @@
+import { FeatureFilterModel } from '../feature-filter.model';
+import { TargetingFilterParameters } from './targeting-feature-filter-parameters';
+import { TargetingFilterAppContext } from './targeting-filter-app-context';
+
+export class TargetingFeatureFilterModel implements FeatureFilterModel {
+
+  name = 'Microsoft.Targeting';
```

Comment:
Did not you define a enum for this value? `FeatureFilterTypes`

Reply:
My bad. Let me update it